src/pages/task/actionButtons/relevantCommits/index.tsx

```diff
@@ -85,6 +86,33 @@ export const PreviousCommits: React.FC<PreviousCommitsProps> = ({ taskId }) => {
     },
   });
   const lastPassingTask = getTaskFromMainlineCommitsQuery(lastPassingTaskData);
+  const passingOrderNumber = lastPassingTask?.order;
+
+  // The breaking commit is the first failing commit after the last passing commit.
+  // The skip order number should be the last passing commit's order number + 1.
+  // We use + 2 because internally the query does a less than comparison.
```

Comment:
I would suggest to leave a permalink here to the GitHub line: 
````
We use + 2 to work around the underlying query which uses the $lt operator (link here).
````

Reply:
I added a link but please let me know if I did it right! I used the HEAD commit currently of the current branch so it's unchanging in the future.